src/transformers/modeling_utils.py

```diff
@@ -932,6 +933,21 @@ def prune_heads(self, heads_to_prune: Dict[int, List[int]]):
 
         self.base_model._prune_heads(heads_to_prune)
 
+    def gradient_checkpointing_enable(self, flag: bool = True):
```

Comment:
Should there be a disable too?

Reply:
Ah I didn't see this had a flag! Maybe `toggle` then? Or `set_gradient_checkpointing` to follow traditional boolean setter conventions?

Reply:
@stas00 really wanted the method name to start with `gradient_checkpointing` to be more easily discoverable.